src/main/java/gregtech/common/ConfigHolder.java

```diff
@@ -158,6 +158,9 @@
         @Config.Comment({"Whether to make coloring blocks like Concrete or Glass harder.", "Default: false"})
         public boolean hardDyeRecipes = false;
 
+        @Config.Comment({"Whether to remove charcoal smelting recipes from the vanilla furnace.", "Default: true"})
+        public boolean harderCharcoalRecipe = true;
```

Comment:
I think this should default to false.

Reply:
I firmly believe that this should default to true. Having it default to false means that there is no real reason to use the coke oven besides the bare minimum to get creosote for the primitive pump. If the player has other mods installed in the pack that adds an early game water source, there is no reason to make the coke oven at all.

Considering that the coke oven is one of the main goals of early game, not having to make it at all, or making it and using it only a couple times makes the entire structure seem like a waste